tests/ekore/anomalous_dimensions/unpolarized/time_like/test_init.py

```diff
@@ -2,9 +2,8 @@
 
 import ekore.anomalous_dimensions.unpolarized.time_like as ad_ut
 
-
-def test_init():
-    with pytest.raises(NotImplementedError):
-        ad_ut.gamma_ns((1, 0), 0, 1.0, 4)
-    with pytest.raises(NotImplementedError):
-        ad_ut.gamma_singlet((1, 0), 1.0, 4)
+# def test_init():
```

Comment:
Either you keep it, or you drop it.

I guess now this has been implemented, so the commented test is outdated. Please, replace with suitable tests.

Reply:
Yes, I will definitely fix this but first I would like to sort the above issue. Do you have any suggestions on why even deleting the pycache doesn't force recompile?